docs/tutorials/tutorial-1.rst

```diff
@@ -0,0 +1,175 @@
+
+.. code-block:: python
+
+    print("Hello World!")
+
+Then click on the "Run code" button. The page will reload, and below the area
+on the page named "Console output", you'll see the output you'd
+expect from this Python program::
+
+    Hello World!
+
+Congratulations! You've just run your first Python program under Javascript
+using Batavia! Now you can get a little more adventurous and try a loop. Replace
+your existing code in the text box with the following:
+
+.. code-block:: python
+
+    for i in range(0, 10):
+        print("Hello World %d!" % i)
+
+Click "Run code" again, and you should see the following on the screen in the
+console output section::
+
+    Hello World 0!
+    Hello World 1!
+    Hello World 2!
+    Hello World 3!
+    Hello World 4!
+    Hello World 5!
+    Hello World 6!
+    Hello World 7!
+    Hello World 8!
+    Hello World 9!
+
+
+What just happened?
+-------------------
+
+What happened when you pressed the "Run code" button?
+
+When you clicked "Run code", your browser submitted the content of the text
+box as a HTTP ``POST`` request to the test server. The test server took that
+content, and compiled it as if it were Python code. It didn't *run* the code --
+it just compiled it to bytecode. It created the ``.pyc`` file that
+would have been produced if you had written the same code into a ``test.py`` file and
+ran ``python test.py``.
+
+Having compiled the source code into bytecode form, it then encoded the
+contents of the ``.pyc`` file into base64, and inserted that base64 string into the
+returned HTML response. If you inspect the source code for the page, you
+should see a block in the document ``<head>`` that looks something like:
+
+.. code-block:: html
+
+    <script id="batavia-customcode" type="application/python-bytecode">
+        7gwNCkIUE1cWAAAA4wAAAAAAAAAAAAAAAAIAAABAAAAAcw4AAABlAABkAACDAQABZAEAUykCegtI
+        ZWxsbyBXb3JsZE4pAdoFcHJpbnSpAHICAAAAcgIAAAD6PC92YXIvZm9sZGVycy85cC9uenY0MGxf
+        OTc0ZGRocDFoZnJjY2JwdzgwMDAwZ24vVC90bXB4amMzZXJyddoIPG1vZHVsZT4BAAAAcwAAAAA=
+    </script>
+
+That string is the base64 encoded version of the Python program you submitted.
+The browser then takes this base64 string, decodes it back into a bytestring,
+and runs it through Batavia -- a 15kb Javascript module that does the same thing
+in a browser that CPython does on the desktop: iterprets Python bytecode as a
+running program.
+
+Push the button...
+------------------
+
+You may also have noticed a set of buttons on the left hand side of the
+screen. These are some pre-canned example code, ready for testing. Try
+clicking the "Run sample.py" button. Your browser should pop
+up a new window and load the `BeeWare website`_. If you close that window and
+go back to the Batavia testbed, you should see a lot of output in the console
+section of the screen.
+
+.. _BeeWare website: http://pybee.org
+
+If you want to, you can `inspect the source code`_. One part of
```

Comment:
is this section related to the "push the button section", or a separate task? maybe separate into a standalone section with a heading